src/sections/Users/index.js

```diff
@@ -1,12 +1,34 @@
 import { useEffect, useState } from "react"
 
 function UsersSection() {
+const [usersData, setUsersData] =  useState([]);
+
+ useEffect(() => {
+ getData();
+}, []);
+
+ async function getData() {
+    const response = await fetch("https://randomuser.me/api/");
+    const json = await response.json();
+    setUsersData(json);
+    console.log(json);
+  }
```

Comment:
The API call is correct, if we check the response we get something like this:

```json
{"results":[{"gender":"female","name":{"title":"Miss","first":"Vanessa","last":"Meehan"},"location":{"street":{"number":2940,"name":"Church Road"},"city":"Passage West","state":"Kildare","country":"Ireland","postcode":21073,"coordinates":{"latitude":"3.3787","longitude":"172.5529"},"timezone":{"offset":"-10:00","description":"Hawaii"}},"email":"vanessa.meehan@example.com","login":{"uuid":"f91141b6-13c7-460f-9504-f14f26524d0a","username":"crazywolf183","password":"4121","salt":"oXCAd9vt","md5":"2b985a21fc0aa4572f66eb72cde6b354","sha1":"91be2dfed9d2909729b590cb1f1fd7bc27d82925","sha256":"c15c92b298a4e6f510d5487e0eeaabb5d1892353bfa212846c4a0cabbdcca4c7"},"dob":{"date":"1960-10-01T22:43:12.647Z","age":62},"registered":{"date":"2021-05-17T04:01:48.411Z","age":2},"phone":"031-312-8060","cell":"081-177-6216","id":{"name":"PPS","value":"3562636T"},"picture":{"large":"https://randomuser.me/api/portraits/women/16.jpg","medium":"https://randomuser.me/api/portraits/med/women/16.jpg","thumbnail":"https://randomuser.me/api/portraits/thumb/women/16.jpg"},"nat":"IE"}],"info":{"seed":"b8b2b849d35e23e9","results":1,"page":1,"version":"1.4"}}
```

This response is an object that has a `results` property. So your `json` variable contains that object which has a property on it called `results`. If you want to map over the data, you'll need to set your user data state variable to `json.results`. Good work, you were almost there :)